components/builder-originsrv/src/migrations/2018-03-05-224023_origin_secrets/up.sql

```diff
@@ -0,0 +1,53 @@
+CREATE SEQUENCE IF NOT EXISTS origin_secrets_id_seq;
+
+CREATE TABLE IF NOT EXISTS origin_secrets (
+  id bigint PRIMARY KEY DEFAULT next_id_v1('origin_secrets_id_seq'),
+  origin_id bigint REFERENCES origins(id),
+  owner_id bigint,
+  name varchar(256),
```

Comment:
I think we should be using varchar in more places. Using text doesn't make sense for things like names.

Reply:
Hmm, why is that? It seems we're getting more flexibility with text, with basically same cost under the covers. Another thing that might be weird is comparison between text and varchars, looks like there are subtle differences in implicit comparison.  It might be better to just be consistent with the rest of the code for now... unless there's some other strong reason?